scripts/regen-py.pl

```diff
@@ -114,7 +114,7 @@ sub needs_method_arg {
     return 1;
 }
 
-emit_functions($ENV{BINARYCOM_API_SCHEMA_PATH} // '/home/git/binary-com/deriv-developers-portal/config/v3');
+emit_functions($ENV{BINARYCOM_API_SCHEMA_PATH} // '/home/git/deriv-com/deriv-developers-portal/config/v3');
```

Comment:
We dont have this repo in deriv-com and the one in binary-com is already archived. should this still exists?

Reply:
it should be deriv-api-docs